teuthology/suite/placeholder.py

```diff
@@ -100,6 +100,9 @@ def _substitute(input_dict, values_dict):
             'sha1': Placeholder('suite_hash'),
         }
     },
+    'syslog': {
+        'ignorelist': [],
+    },
```

Comment:
Why is this bit needed?  `placeholder.py` generally seems to encode non-default values.  Wouldn't an empty `ignorelist` be the default anyway?